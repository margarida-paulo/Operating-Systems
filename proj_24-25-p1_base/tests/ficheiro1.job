HELP
SHOW
WRITE [(umnome,umvalor)(outronome,outrovalor)]
SHOW
