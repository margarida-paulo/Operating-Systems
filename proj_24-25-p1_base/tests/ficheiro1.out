Available commands:
  WRITE [(key,value)(key2,value2),...]
  READ [key,key2,...]
  DELETE [key,key2,...]
  SHOW
  WAIT <delay_ms>
  BACKUP
  HELP
(Fernando,Pessoa)
(Luís,Camões)
(Mariana,Mari)
